addons/utm/models/utm.py

```diff
@@ -1,86 +1,50 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.osv import osv, fields
-from openerp.http import request
+from openerp import fields, models
 
 
-class utm_medium(osv.Model):
+class UtmMedium(models.Model):
     # OLD crm.case.channel
-    _name = "utm.medium"
-    _description = "Channels"
+    _name = 'utm.medium'
+    _description = 'Channels'
     _order = 'name'
-    _columns = {
-        'name': fields.char('Channel Name', required=True),
-        'active': fields.boolean('Active'),
-    }
-    _defaults = {
-        'active': lambda *a: 1,
-    }
 
+    name = fields.Char(string='Channel Name', required=True)
+    active = fields.Boolean(string='Active', default=True)
```

Comment:
Dont give string when string is same as variable name because when you not given framework automatically take name as string with Camelcase